mods/d2k/chrome/ingame-leavemap.yaml

```diff
@@ -103,6 +110,13 @@ Container@LEAVE_MAP_WIDGET:
 							Height: 25
 							Font: Bold
 							Text: Chat
+				Button@RESTART_BUTTON:
+					X: PARENT_RIGHT - 3 * (WIDTH + 20)
```

Comment:
`X: 20`, please. It's looks better when the button is all the way to the left side.